features/in_onderzoek.feature

```diff
@@ -166,188 +165,283 @@ Functionaliteit: in onderzoek
+      | fields              | verblijfplaats.verblijfadres.<veld> |
+      Dan heeft de response een persoon met de volgende 'verblijfplaats' gegevens
+      | naam                                                       | waarde              |
+      | type                                                       | Locatie             |
+      | verblijfadres.locatiebeschrijving                          | locatiebeschrijving |
+      | verblijfadres.inOnderzoek.<veld>                           | true                |
+      | verblijfadres.inOnderzoek.datumIngangOnderzoek.datum       | 2022-03-07          |
+      | verblijfadres.inOnderzoek.datumIngangOnderzoek.type        | Datum               |
+      | verblijfadres.inOnderzoek.datumIngangOnderzoek.langFormaat | 7 maart 2022        |
+
+      Voorbeelden:
+      | waarde | veld                |
+      | 081210 | locatiebeschrijving |
+
+    Scenario: persoon heeft land van verblijfplaats in onderzoek
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 555550001 |
+      En de persoon heeft de volgende 'verblijfplaats' gegevens
+      | naam                                     | waarde   |
+      | land adres buitenland (13.10)            | 1234     |
+      | aanduiding gegevens in onderzoek (83.10) | 081310   |
+      | datum ingang onderzoek (83.20)           | 20220307 |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                            |
+      | type                | RaadpleegMetBurgerservicenummer   |
+      | burgerservicenummer | 555550001                         |
+      | fields              | verblijfplaats.verblijfadres.land |
+      Dan heeft de response een persoon met de volgende 'verblijfplaats' gegevens
+      | naam                                                       | waarde                   |
+      | type                                                       | VerblijfplaatsBuitenland |
+      | verblijfadres.land.code                                    | 1234                     |
+      | verblijfadres.inOnderzoek.land                             | true                     |
+      | verblijfadres.inOnderzoek.datumIngangOnderzoek.datum       | 2022-03-07               |
+      | verblijfadres.inOnderzoek.datumIngangOnderzoek.type        | Datum                    |
+      | verblijfadres.inOnderzoek.datumIngangOnderzoek.langFormaat | 7 maart 2022             |
+
+
+    Scenario: persoon heeft regel3 van verblijfplaats in onderzoek
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 555550001 |
+      En de persoon heeft de volgende 'verblijfplaats' gegevens
+      | naam                                     | waarde   |
+      | land adres buitenland (13.10)            | 1234     |
```

Comment:
Moet dit niet 

| Regel 3 adres buitenland (13.50) | xxxx |

zijn (met voor xxxx een reële waarde)? In regel 309 verwijs je immers naar 081350.

Reply:
Hier is trouwens ook sprake van inconsistentie tussen de scenario's. In de tabel van regel 329 t/m 331 neem je nl. geen waarde voor 'Aanduiding verblijfstitel (39.10)' op.

Reply:
Die scenario beschrijft de situatie dat een verblijfplaats geen regel 3 gegeven heeft, maar dat  en dat dit gegeven wel in onderzoek staat. Dat is wat ik in de initiele scenario heb geïnterpreteerd

Reply:
> Hier is trouwens ook sprake van inconsistentie tussen de scenario's. In de tabel van regel 329 t/m 331 neem je nl. geen waarde voor 'Aanduiding verblijfstitel (39.10)' op.

Dat er geen 'aanduiding verblijftitel (39.10)' is, kan de reden zijn dat het in onderzoek staat

Reply:
> Die scenario beschrijft de situatie dat een verblijfplaats geen regel 3 gegeven heeft, maar dat en dat dit gegeven wel in onderzoek staat. Dat is wat ik in de initiele scenario heb geïnterpreteerd

Dat zou ik dan in de beschrijving van het scenario aangeven.